docs/configuration-github.md

```diff
@@ -35,37 +35,44 @@ The App should also subscribe to the following webhook events:
 
 Description, Homepage, User authorization callback URL, and Setup URL are all unimportant so you may set them to whatever you like.
 
-The Mend Renovate webhook listener binds to port 8080 by default, however it will bind to `process.env.PORT` instead if that is defined.
-Note: The Mend Renovate image takes care of exposing port 8080 of the container, so if you change this port then you will need to take care of any exposing/mapping of ports yourself.
-In the [Docker Compose example config](https://github.com/mend/renovate-on-prem/tree/main/examples/), the default port 8080 is used but then mapped to port 80 on the host.
+The Mend Renovate webhook listener binds to port 3000 by default, however it will bind to `process.env.PORT` instead if that is defined.
+Note: The Mend Renovate image takes care of exposing port 3000 of the container, so if you change this port then you will need to take care of any exposing/mapping of ports yourself.
+In the [Docker Compose example config](https://github.com/mend/renovate-on-prem/tree/main/examples/), the default port 3000 is used and then mapped to port 3000 on the host.
```

Comment:
```suggestion
The Mend Renovate webhook listener binds to port 8080 by default, however it will bind to `process.env.PORT` instead if that is defined.
Note: The Mend Renovate image takes care of exposing port 8080 of the container, so if you change this port then you will need to take care of any exposing/mapping of ports yourself.
In the [Docker Compose example config](https://github.com/mend/renovate-on-prem/tree/main/examples/), the default port 8080 is used and then mapped to port 80 on the host.
```